dist/tools/codespell/check.sh

```diff
@@ -38,7 +38,7 @@ ${CODESPELL_CMD} --version &> /dev/null || {
 CODESPELL_OPTS="-q 2"  # Disable "WARNING: Binary file"
 CODESPELL_OPTS+=" --check-hidden"
 # Disable false positives "nd  => and, 2nd", "WAN => WANT", "od => of"
-CODESPELL_OPTS+=" --ignore-words-list=ND,nd,WAN,od"
+CODESPELL_OPTS+=" --ignore-words-list=ND,nd,WAN,od,DOUT"
```

Comment:
Codespell errors won't block a PR, no need to add an exception.

Reply:
Sure, that makes sense now.